media-video/gnash/gnash-0.8.11~git.recipe

```diff
@@ -86,7 +86,7 @@ REQUIRES_devel="
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libagg$secondaryArchSuffix
-	devel:libavcodec$secondaryArchSuffix <= 58.54.100
+	devel:libavcodec$secondaryArchSuffix <= 58
```

Comment:
```suggestion
	devel:libavcodec$secondaryArchSuffix >= 58
```

Reply:
@korlu - That change triggers ffmpeg5. Gnash not adapted to ffmpeg5.